pkg/controllers/monitoring-stack/controller.go

```diff
@@ -196,3 +226,35 @@ func (r *reconciler) reconcileObject(ctx context.Context, ms *stack.MonitoringSt
 	logger.Info("Updating stack component")
 	return r.k8sClient.Update(ctx, desired)
 }
+
+func (r *reconciler) createGrafanaDSWatch(ctx context.Context) (bool, error) {
+	if r.grafanaDSWatchCreated {
+		return false, nil
+	}
+	log := r.logger.WithName("create-grafana-ds-watch")
+	var dataSources grafanav1alpha1.GrafanaDataSourceList
+
+	err := r.k8sClient.List(ctx, &dataSources, client.InNamespace("default"))
+
+	if err != nil {
+		log.V(3).Info("grafana data source CRD is not defined")
```

Comment:
IIRC, @fpetkovski changed these log levels to `V(6)` so you may want to keep it the same.